upgrade/upgrade-scripts/upgrade-container

```diff
@@ -428,6 +428,12 @@ function migrate_file() {
 			die "'mkdir -p $parentdir' failed for '$CONTAINER'"
 		cp "$path" "/var/lib/machines/${CONTAINER}${path}" ||
 			die "'cp $path' failed for '$CONTAINER'"
+	else
+		# If a migration target file doesn't exist pre-upgrade then
```

Comment:
Can we reword this to something like: "If the target file doesn't exist" .. it's minor, but since we use "migration" as the name for illumos to linux upgrades, it can be a little confusing to call the file a "migration target file" (e.g. if one assumes "migration" used here means "illumos to linux migration").